src/App.tsx

```diff
@@ -1,10 +1,40 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
+
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import { Header } from './components/Header';
+import { Main } from './components/Main';
+import { Footer } from './components/Footer';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
 
-const USER_ID = 0;
+const USER_ID = 52;
 
 export const App: React.FC = () => {
+  const [posts, setPosts] = useState<Todo[]>([]);
+  const [errorLoad, setErrorLoad] = useState<boolean>(false);
+  const [filter, setFilter] = useState('all');
```

Comment:
```suggestion
  const [filter, setFilter] = useState<Filter>(Filter.All);
```
Move `Filter` enum into a separate file (`types.ts` for example) and export it. Import everywhere you want to use it